tests/test_octodns_provider_ns1.py

```diff
@@ -1092,6 +1135,20 @@ def test_data_for_dynamic_A(self):
         data2 = provider._data_for_A('A', ns1_record)
         self.assertEquals(data, data2)
 
+        # Oceania test cases
+        # 1. Full list of countries should return 'OC' in geos
+        oc_country_list = self._CONTINENT_TO_LIST_OF_COUNTRIES['OC']
+        ns1_record['regions']['lhr']['meta']['country'] = oc_country_list
+        data3 = provider._data_for_A('A', ns1_record)
+        assert('OC' in data3['dynamic']['rules'][0]['geos'])
```

Comment:
I probably should add another assert here that none of the 'OC' countries are present under the 'country' tag.

Reply:
In tests you'll want to use `self.assert*` rather than bare `assert` so that you get all the unittest features/details.

```suggestion
        self.assertTrue('OC' in data3['dynamic']['rules'][0]['geos'])
```